cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -103,5 +101,12 @@ spec:
       - name: lib-modules
         hostPath:
           path: /lib/modules
+      tolerations:
+      - key: CriticalAddonsOnly
+        operator: Exists
+      - key: {{ .MasterTaintKey }}
+        effect: NoSchedule
+      nodeSelector:
+        beta.kubernetes.io/arch: {{ .Arch }}
```

Comment:
Should this have a TODO comment to remove after manifest list images are available?